scenariolib/event_fakeSearch.go

```diff
@@ -0,0 +1,44 @@
+// Package scenariolib handles everything need to execute a scenario and send all
+// information to the usage analytics endpoint
+package scenariolib
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/coveo/go-coveo/search"
+)
+
+// ============== FAKE SEARCH EVENT ======================
+// ==================================================
+
+// FakeSearchEvent a struct representing a search, is the response to set in the visit
+type FakeSearchEvent struct {
+	fakeResponse *search.Response
+}
+
+func newFakeSearchEvent(e *JSONEvent, c *Config) (*FakeSearchEvent, error) {
```

Comment:
No need to comment an unexposed function that is pretty self explanatory in the title.
You can if you want @adambbolduc but this won't block the merge.